File: beginner/javascript/2160.js
const readline = require('readline');

const rl = readline.createInterface({
  input: process.stdin,
  output: process.stdout
});

rl.question('', (texto) => {
  if (texto.length < 81) {
    console.log("YES");
  } else {
    console.log("NO");
  }
  rl.close();
});

File: beginner/java/2160.java
import java.util.Scanner;

public class Main {
    public static void main(String[] args) {
        Scanner scanner = new Scanner(System.in);

        String texto = scanner.nextLine();

        if (texto.length() < 81) {
            System.out.println("YES");
        } else {
            System.out.println("NO");
        }

        scanner.close();
    }
}
